R/create_realistic_org.R

```diff
@@ -67,7 +67,7 @@ create_realistic_org <- function(n_children = 4, max_depth = 3, prob=0.3, .f=NUL
   }
 
   out <- create_regular_org(n_children, max_depth) %>%
-    tidygraph::mutate(unit_id = dplyr::row_number(),
+    tidygraph::mutate(unit_id = dplyr::row_number() %>% as.character(),
```

Comment:
why as.character? int comparison will be faster; is it for plotting reasons?

Reply:
I realised that unit id's will often not be integers - eg in the people survey they take the form DWP0123.